### PR TITLE
Add correct testing idx for segmentation tutorial

### PR DESCRIPTION
**Pull request by ulya-tkch**

### CODE REVIEW

File: docs/source/tutorials/segmentation.ipynb [hunk 1]
```diff
@@ -449,13 +449,13 @@
...
+    "assert (top_2_issues == [1,  21]).all()\n",
     "\n",
     "top_3_class_issues = np.argsort(-np.sum(class_issues, axis=(1, 2)))[:3]\n",
-    "assert (top_3_class_issues == [134,  66, 109]).all()\n",
+    "assert (top_3_class_issues == [17,  19, 0]).all()\n",
     "\n",
-    "highlighted_indices = [ 78,  16, 120,  34, 137,  51,  71,   8, 106,  94]\n",
-    "if not all(np.argsort(issues_from_score.sum((1,2)))[:10]==highlighted_indices):\n",
+    "highlighted_indices = [ 1,  21, 2,  24, 4, 3, 12]\n",
+    "if not all(np.argsort(-issues_from_score.sum((1,2)))[:7]==highlighted_indices):\n",
```
Comment (jwmueller) [inline docs/source/tutorials/segmentation.ipynb:458]:
```suggestion
    "if not all(np.argsort(-issues_from_score.sum((1,2)))[:len(highlighted_indices)]==highlighted_indices):\n",
```

Comment (jwmueller) [inline docs/source/tutorials/segmentation.ipynb:458]: just curious: why do all the argsorts require negation before them?

File: docs/source/tutorials/segmentation.ipynb [hunk 2]
```diff
@@ -449,13 +449,13 @@
    "source": [
```
Comment (jwmueller) [inline docs/source/tutorials/segmentation.ipynb:449]: need to resolve merge conflict too, think you made this PR from an outdated starting state

File: docs/source/tutorials/segmentation.ipynb (hunk 1 above)
Comment (ulya-tkch) [reply docs/source/tutorials/segmentation.ipynb:458]: `issues_from_score.sum((1,2))` returns a sum of the number of issues per image. Therefore images with more pixel issues have higher scores. Argsort sorts an array from lower to highest value so we negate the array prior to sorting to get the top 7 images with the most issues.

**Review by huiwengoh — APPROVED** [review]
Ran this notebook locally and the indices were correct.

However I think this last code block is not actually hidden from docs site. Make sure it's marked as hidden in the metadata ([this line](https://github.com/cleanlab/cleanlab/pull/756/files#diff-1e866a038d3ea940531ab6f4ddcd1dd2fcf394f95b988102593ce55566012b1eR447)) and feel free to merge after!  -- Edit: I've added this metadata tag

**Comment by jwmueller** [comment]
> Ran this notebook locally and the indices were correct.
> 
> However I think this last code block is not actually hidden from docs site. Make sure it's marked as hidden in the metadata ([this line](https://github.com/cleanlab/cleanlab/pull/756/files#diff-1e866a038d3ea940531ab6f4ddcd1dd2fcf394f95b988102593ce55566012b1eR447)) and feel free to merge after! -- Edit: I've added this metadata tag and will merge!

Great catch @huiwengoh !  Ulyana still has some other updates pending for this PR regarding some more tests and confusing docstrings, so I'll handle merge after those are in

**Pull request merged**